Source/Scene/ShadowMap.js

```diff
@@ -107,37 +107,28 @@ define([
 
     /**
      * Creates a shadow map from the provided light camera.
+     * <p>
+     *     Use {@link Viewer#shadowMap}
```

Comment:
It's fine to remove the indentation for this line.

Expand on the description a bit. Maybe `Use {@link Viewer#shadowMap} to get the scene's shadow map originating from the sun. In general do not construct directly.`